thundernest/k-9
# Change `EmailAddressParser` to support addresses violating the specification

Steps to test:
1. Run a debug build
2. Add an account for `demo@k9mail.example` (contains a message with an email address where the local part exceeds the 64 character limit)
3. Enable *Settings → General settings → Display → Show contact names*
4. Loading the message list for the inbox fails without this change and should load fine with this change applied.

Fixes #6980
